pycontrails/models/cocip/cocip_params.py

```diff
@@ -136,6 +136,15 @@ class CocipParams(ModelParams):
     #: :attr:`CocipGridParams.verbose_outputs_evolution`.
     verbose_outputs: bool = False
 
+    #: Add additional metric of ATR20 and global mean RF to model output.
+    #: These are not standard CoCiP outputs but based on the derivation used
+    #: by :cite:`dietmullerPythonLibraryComputing2022`.
+    compute_atr20: bool = False
+
+    #: Constant factor used to convert global mean RF, [:math:`W m^{-2}`],
+    #: to ATR20, [:math:`K`], given by :cite:`dietmullerPythonLibraryComputing2022`.
+    global_rf_to_atr20_factor: float = 0.0151
+
```

Comment:
Similarly, for the sake of clarity I think it's worth changing "global mean RF" to something like "global- and year-mean contrail RF".

And, maybe it's worth adding a little bit more information about what ATR20 is? (At least a definition, e.g. "an estimate of the 20-year-average temperature change caused by the radiative forcing of a contrail", if that's accurate.)